README.md

```diff
@@ -478,8 +507,8 @@ Offline Installations   | ✓     | x
   For example, `npm` allows `file` dependencies with the `cachito_npm_file_deps_allowlist`
   configuration. `gomod` allows this through the `go.mod` replace directive.
 * **Offline Installations** - The dependencies can be installed solely with the contents of the
-  bundle. This is true for the `gomod` package manager, however, the `npm` package manager relies
-  on Nexus to be online and properly configured by Cachito.
+  bundle. This is true for the `gomod` package manager, however, the `npm` and `pip` package
+  managers rely on Nexus to be online and properly configured by Cachito.
```

Comment:
(Optional) Should we give an example for pip for `External Dependency` on https://github.com/release-engineering/cachito/pull/273/files#diff-04c6e90faac2675aa89e2176d2eec7d8R501?

Reply:
IMO that is not necessary, I'd say the npm example is generic enough to also apply to pip (and potentially other package managers in the future)